CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 
 ## Pending
+- Breaking Change: #140 (ark-poly) Move methods `from_coefficients_slice`, `from_coefficients_vec`, `terms` from `MVPolynomial` to `MVPolynomialCoefficientForm`
+- Feature: #140 (ark-poly) Add support for multilinear polynomial in dense and sparse evaluation form.
```

Comment:
Can you remove these two lines from here?

Reply:
resolved